src/compiler/constraints.jl

```diff
@@ -77,17 +131,17 @@ function toqubo_constraint(
 end
 
 function toqubo_constraint(
-    model::VirtualQUBOModel{T},
+    model::VirtualModel{T},
     f::SQF{T},
     s::EQ{T},
     arch::AbstractArchitecture,
 ) where {T}
-    # -*- Scalar Quadratic Function: g(x) = x Q x + a x - b = 0 😢 -*-
+    # Scalar Quadratic Function: g(x) = x Q x + a x - b = 0
```

Comment:
I would replace functions by equality constraints to differentiate them from the inqualities